<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vik ВКонтакте</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="friends">
     <attribute name="icon">
      <iconset resource="ux_icons.qrc">
       <normaloff>:/icons/tabs/friends.png</normaloff>:/icons/tabs/friends.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
    <widget class="QWidget" name="message">
     <attribute name="icon">
      <iconset resource="ux_icons.qrc">
       <normaloff>:/icons/tabs/chat.png</normaloff>:/icons/tabs/chat.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QWidget" name="dialogs" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>421</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="messages" native="true">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>421</width>
        <height>371</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="text">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>390</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="send">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>390</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>626</x>
        <y>390</y>
        <width>31</width>
        <height>26</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ux_icons.qrc">
        <normaloff>:/icons/buttons/smile-button.png</normaloff>:/icons/buttons/smile-button.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="audio">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#9200a8;&quot;&gt;Мои аудиозаписи&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <attribute name="icon">
      <iconset resource="ux_icons.qrc">
       <normaloff>:/icons/tabs/music-notes.png</normaloff>:/icons/tabs/music-notes.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
    <widget class="QWidget" name="downloads">
     <attribute name="title">
      <string>Загрузки</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="accessToken">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>531</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="userId">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpt">
    <property name="title">
     <string>Opt</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuOpt"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Переавторизовать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ux_icons.qrc"/>
 </resources>
 <connections/>
</ui>
